hack/generate/csv.sh

```diff
@@ -88,6 +88,7 @@ kafka_image "kafka-ch-controller__controller"      "${eventing_kafka}-consolidat
 kafka_image "DISPATCHER_IMAGE"                     "${eventing_kafka}-consolidated-dispatcher"
 kafka_image "kafka-ch-dispatcher__dispatcher"      "${eventing_kafka}-consolidated-dispatcher"
 kafka_image "kafka-webhook__kafka-webhook"         "${eventing_kafka}-webhook"
+kafka_image "storage-version-migration-kafka-kafka-$(metadata.get dependencies.eventing_kafka)__migrate" "${eventing_kafka}-storage-version-migration"
```

Comment:
I don't think this is correct as the job is named `v0.21-kafka-storage-version-migration`, so I'd think this override will have to be

```suggestion
kafka_image "v0.21-kafka-storage-version-migration__migrate" "${eventing_kafka}-storage-version-migration"
```

Reply:
ah. yea - the name of the job is `v0.21-kafka-storage-version-migration` - thanks